pkgs/development/compilers/swift/default.nix

```diff
@@ -27,6 +32,15 @@ let
         llvmPackages_15.clang.override rec {
           libc = apple_sdk.Libsystem;
           bintools = pkgs.bintools.override { inherit libc; };
+          # Ensure that Swift’s internal clang is using the same libc++ and libc++abi
+          # used by the default Darwin stdenv.
```

Comment:
Can we expand this comment briefly to add what goes wrong if we don't do this?

Reply:
I expanded the comment and also added one for compiler-rt.